cypress-tests/cypress/e2e/admin/pageBuilder/templates/useBlankPageTemplate.cy.ts

```diff
@@ -0,0 +1,17 @@
+context("Page Builder - Blocks", () => {
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+    });
+
+    it("Should be able to create a page and view all existing templates in it", () => {
+        cy.visit("/page-builder/pages?folderId=root");
+        cy.findByTestId("new-page-button").click();
+
+        cy.contains("Pick a template for your new page").should("exist");
+
+        cy.get("button.webiny-ui-button--secondary").eq(1).click();
```

Comment:
Let's do a test ID for this. Please attach a screenshot here and I'll add it.

Reply:
![image](https://github.com/webiny/webiny-js/assets/35311113/a456951e-c015-4854-b9f2-644edead8883)
The button on the top right and bottom left

Reply:
`pb-pages-list-use-blank-template-btn` and `pb-pages-list-use-template-btn`